features/attach_validtoken.feature

```diff
@@ -26,13 +51,35 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
         """
         Enabling default service esm-infra
         """
+        When I run `/usr/lib/update-notifier/apt-check  --human-readable` as non-root
+        Then if `<release>` in `trusty` and stdout matches regexp:
+        """
+        UA Infrastructure Extended Security Maintenance \(ESM\) is enabled.
 
+        \d+ updates can be installed immediately.
+        \d+ of these updates are provided through UA Infrastructure ESM.
+        \d+ of these updates are security updates.
+        To see these additional updates run: apt list --upgradable
+        """
+        Then if `<release>` in `xenial||bionic` and stdout matches regexp:
+        """
+        \d+ packages can be updated.
```

Comment:
I if missed something in the discussion here, but why are xenial and bionic not fully covered by ESM ?

Also, is the check regarding that we should only send a message if there is a package to be upgraded be delivered in a separate PR ?

Reply:
I was thinking we would need a separate PR to validate whether or not we see the message if there are no esm packages, but maybe we should just refactor this PR to do that as I would like to close out the ESM motd issue fully.

Reply:
Ahh ok, so this is this issue that I just filed to track it. There has been limited SRUing of update-notifier functionality which would surface ESM-aware apt_check.py --human-readable. 
We will probably have to shepherd this work and maybe work with either andreas or bryce on pkg upload sponsorship.